rbkit-lib/model/cpustorage.cpp

```diff
@@ -0,0 +1,141 @@
+#include "cpustorage.h"
+#include <QVariant>
+#include <QDebug>
+#include "cpumapping.h"
+
+void CpuStorage::addNewNode(QMap<int, QVariant> data)
+{
+    //qDebug() << data;
+    CpuNodePtr newNode( new CpuNode(data[RBKit::CeMethodName].toString(),
+                data[RBKit::CeLabel].toString(),
+                data[RBKit::CeFile].toString(),
+                data[RBKit::CeThreadId].toString(),
+                data[RBKit::CeLine].toDouble(),
+                data[RBKit::CeSingletonMethod].toInt())
+            );
+
+    CpuStorage::updateNewNodeLocation(data[RBKit::CeMethodName].toString(), newNode);
```

Comment:
okie, then create temp variables in this method, at the beginning.

```c++
  auto methodName = data[RBKit::CeMethodName].toString()
  // ...
  CpuNodePtr node(new CpuNode(methodName, ...));
```

something like this.